pkg/apiclient/grpcproxy.go

```diff
@@ -113,7 +114,19 @@ func (c *client) startGRPCProxy() (*grpc.Server, net.Listener, error) {
 			if err != nil {
 				return err
 			}
+
 			md, _ := metadata.FromIncomingContext(stream.Context())
+
+			if c.AdditionalHeaders != "" {
+				for _, kv := range strings.Split(c.AdditionalHeaders, ",") {
```

Comment:
See comment about using `StringVarArray` and passing down an array instead.

Reply:
Used StringSliceVar in latest commit